### PR TITLE
[10.0][FIX] queue_job:Issue when create channel from UI.

### PR DESCRIPTION
**Pull request by vjd8866**
- [ FIX ]

### CODE REVIEW

**Comment by guewen** [comment]
Hi,

Thanks for the fix! I think a better fix would be to have `required=True` on the name field.

**Review by guewen — CHANGES_REQUESTED** [review]
The correct solution is to add a required on the name

**Comment by guewen** [comment]
> The correct solution is to add a required on the name

Wrong :-/

@vdo-serpentcs did you sign the CLA?

File: queue_job/models/queue_job.py [hunk 1]
```diff
@@ -267,8 +267,8 @@ class JobChannel(models.Model):
     @api.depends('name', 'parent_id.complete_name')
     def _compute_complete_name(self):
         for record in self:
-            # if not record.name:
-            #     return  # new record
+            if not record.name:
+                return  # new record
```
Comment (pedrobaeza) [inline queue_job/models/queue_job.py:271]: Should it be `continue`? Or better to use `self.filtered('name')` in the loop

Comment (guewen) [reply queue_job/models/queue_job.py:271]: Good catch :) (however continue is better than `self.filtered` as anyway have to loop on the records, `filtered` would use 2 loops)

**Comment by guewen** [comment]
Considering the change needed and my uncertainty on the CLA, I'm opening #95.

Thanks @vdo-serpentcs!

**Comment by vjd8866** [comment]
@guewen No i have not signed CLA

**Comment by vjd8866** [comment]
> > The correct solution is to add a required on the name
> 
> Wrong :-/
> 
> @vdo-serpentcs did you sign the CLA?

@guewen I have signed CLA but how can I know it's approved, I don't get any confirmation.

**Pull request closed without merging**